api/server/docker.go

```diff
@@ -744,11 +851,15 @@ func (d *driver) unmount(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
-	opts := make(map[string]string)
-	opts[options.OptionsDeleteAfterUnmount] = "true"
-
-	err = v.Unmount(id, mountpoint, opts)
-	if err != nil {
+	mountClient := api.NewOpenStorageMountAttachClient(conn)
+	if _, err = mountClient.Unmount(ctx, &api.SdkVolumeUnmountRequest{
+		VolumeId:  vol.Id,
+		MountPath: mountpoint,
+		Options: &api.SdkVolumeUnmountRequest_Options{
```

Comment:
Lets have a common utility function that converts opts map[string]string into SDK.Unmount.Options